workflow-examples/src/test/java/com/redhat/parodos/notification/integration/NotificationRecordTest.java

```diff
@@ -0,0 +1,135 @@
+package com.redhat.parodos.notification.integration;
+
+import com.redhat.parodos.notification.sdk.api.*;
+import com.redhat.parodos.notification.sdk.model.NotificationMessageCreateRequestDTO;
+import com.redhat.parodos.notification.sdk.model.NotificationRecordResponseDTO;
+import com.redhat.parodos.notification.sdk.model.PageNotificationRecordResponseDTO;
+import com.redhat.parodos.notification.sdk.model.Pageable;
+import com.redhat.parodos.workflow.utils.CredUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+@Slf4j
+public class NotificationRecordTest {
+
+	private static final String BASE_PATH = "http://localhost:8081";
+
+	private NotificationRecordApi recordApiInstance;
+
+	private NotificationMessageApi messageApiInstance;
+
+	@Before
+	public void setUp() throws IOException {
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds("test", "test"));
+		apiClient.setBasePath(BASE_PATH);
+		recordApiInstance = new NotificationRecordApi(apiClient);
+		messageApiInstance = new NotificationMessageApi(apiClient);
+	}
+
+	@Test
+	public void notificationServiceHappyHappyFlow() {
+		String testName = "notificationServiceHappyHappyFlow";
+		logTestStep(testName, "Create the notification messages");
+		createNotificationMessage("type0", "body0", "subject0", "test");
+		createNotificationMessage("type1", "body1", "subject1", "test");
+		createNotificationMessage("type2", "body2", "subject2", "test");
+		createNotificationMessage("type3", "body3", "subject3", "test2");
+
+		validateNotificationRecordCount(testName, 3);
+		PageNotificationRecordResponseDTO list = validateNotificationRecordList(testName, 3);
+
+		NotificationRecordResponseDTO notificationRecord = list.getContent().get(0);
+
+		logTestStep(testName, "Update one Notification Record as \"READ\"");
+
+		try {
```

Comment:
we shouldn't use try in test, should be assertDoesNotThrow

Reply:
`assertDoesNotThrow` is supported only since junit 5.2, while we use junit 4.13.1
As far as I understand, this can't be updated yet because of Maven supprt
Am I wrong?

Reply:
we used assertDoesNotThrow in the workflow-service, are u sure about?

Reply:
Ok, I was looking for [org.junit.Assert.assertDoesNotThrow](https://junit.org/junit5/docs/5.8.2/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable))
Imported `org.junit.jupiter` library to `workflow-examples` as well